includes/elementor-functions.php

```diff
@@ -179,13 +179,14 @@ function hello_get_footer_layout_class() {
  * @return bool
  */
 function hello_get_header_display() {
-	$is_editor = isset( $_GET['elementor-preview'] );
+	if( isset( $_GET['elementor-preview'] ) ) {
+		return true;
+	}
 
 	return (
 		hello_elementor_get_setting( 'hello_header_logo_display' )
 		|| hello_elementor_get_setting( 'hello_header_tagline_display' )
 		|| hello_elementor_get_setting( 'hello_header_menu_display' )
-		|| $is_editor
```

Comment:
What we meant was just to move the $is_editor var to the top of the return statement, not add another one.
Like so:

```
$is_editor = isset( $_GET['elementor-preview'] );

return (
	$is_editor ||
	hello_elementor_get_setting( 'hello_header_logo_display' ) ||
	hello_elementor_get_setting( 'hello_header_tagline_display' ) ||
	hello_elementor_get_setting( 'hello_header_menu_display' )
);
```

Reply:
Fixed